### ISSUE TITLE
Removing tags in video description

### ISSUE THREAD

**Issue by nishkaamprem** [TeamNewPipe/NewPipe#11379]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

So, what im requesting is an removing the tags from the video description completely or an option to remove them such as turning off the "show meta info" button.

Either way is fine with me.

### Why do you want this feature?

1. Removing distraction
2. Looks ugly 
3. #Bloat
4. just because

### Additional information

Basically,
![Imagepipe_23](https://github.com/user-attachments/assets/3c197515-abe8-4b18-b485-baf5623c4b71)

**Comment by exefile2** [comment]
Right
